.github/workflows/tests.yaml

```diff
@@ -72,7 +74,7 @@ jobs:
                   ./scripts/run_in_build_env.sh \
                      "./scripts/build/build_examples.py \
                         --target linux-x64-chip-tool-${BUILD_VARIANT}${CHIP_TOOL_VARIANT} \
-                        --target linux-x64-all-clusters-${BUILD_VARIANT} \
+                        --target linux-x64-all-clusters-${GROUP_VARIANT} \
```

Comment:
should the test-group be a matrix? will we ever have different versions?

The no-ble-tsan is a  matrix because on darwin we already have asan.

how about `--target linux-x64-all-clusters-${BUILD_VARIANT}-test-group` ?

Reply:
Fixed